public/Connect-DbaInstance.ps1

```diff
@@ -917,16 +919,18 @@ function Connect-DbaInstance {
             # But ConnectionContext.IsOpen does not tell the truth if the instance was just shut down
             # And we don't use $server.ConnectionContext.Connect() as this would create a non pooled connection
             # Instead we run a real T-SQL command and just SELECT something to be sure we have a valid connection and let the SMO handle the connection
-            Write-Message -Level Debug -Message "We connect to the instance by running SELECT 'dbatools is opening a new connection'"
             try {
-                $null = $server.ConnectionContext.ExecuteWithResults("SELECT 'dbatools is opening a new connection'")
+                if (-not $DedicatedAdminConnection) {
+                    Write-Message -Level Debug -Message "We connect to the instance by running SELECT 'dbatools is opening a new connection'"
+                    $null = $server.ConnectionContext.ExecuteWithResults("SELECT 'dbatools is opening a new connection'")
+                    Write-Message -Level Debug -Message "We have a connected server object"
+                }
             } catch {
                 if ($DedicatedAdminConnection) {
```

Comment:
We don't need this if we don't execute the query for a DAC.

Reply:
true, ty! ill clean that up